src/services/authorizations-service.ts

```diff
@@ -26,7 +26,11 @@ export class KnexAuthorizationsService implements AuthorizationsService {
 
   async sendAuthorizationsResponse (transactionRequestId: string, request: AuthorizationsIDPutResponse, headers: { [k: string]: string }): Promise<void> {
     this._logger.debug('Authorizations Service: sending Authorizations Response: ' + transactionRequestId)
+    console.log('Inside send authorization'+transactionRequestId)
```

Comment:
can we remove the `console.log` or if you do want them there then use `this._logger`